### DISCUSSION TITLE
some bugs in rustdesk-1.2.0-x86_64-pc-windows-msvc.exe

### DISCUSSION THREAD

**Discussion by fastertogether** [rustdesk/rustdesk#2511] [General]
Thanks too, for this wonderful app.

1. New connection window, the password of local PC is not hidden any more, why?
2. New connection window, the "setting" button, with three dots next to the ID of local PC, no longer pop up menu, but open a "setting" tab, just as when clicked the "global setting" button with three dashes next to the minimize window button. 
3. I like the new top bar comparing with 1.1.9, but there should be a minimize window button, otherwise I've to quit fullscreen so as to switch back to my local PC.
4. afaik, there's no keyboard shortcuts or default setting for switching between fullscreen or window mode.
5. it seems that it freezes the "Alt" or "Win" key more often, when I suddenly cannot type anything now and then at remote PC, and recover after pressing all these control keys.

**How did you install `RustDesk`?**

GitHub release uploaded two hours ago

---

**RustDesk version and environment**

1.2.0 win10

**Comment by fastertogether** [comment]
the freezed key seems to be "ctrl"

**Reply by rustdesk** [reply]
We are working on keyboard.

**Reply by rustdesk** [reply]
Could you submit a seperate issue for this?

**Reply by fastertogether** [reply]
you mean the freezing key issue? sure. Key freezing and no minimize in fullscreen mode cause much trouble these two days. but I haven't tested on latest release
